### PR TITLE
xen: ololo

### PR DESCRIPTION
**Pull request by otyshchenko1**
Signed-off-by: Oleksandr Tyshchenko olekstysh@gmail.com

### CODE REVIEW

**Review by aanisov — CHANGES_REQUESTED** [review]
I would not merge this patch

**Comment by otyshchenko1** [comment]
why?

**Comment by aanisov** [comment]
because!

File: ololo.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+ololo
```
Comment (aanisov) [inline ololo.txt]: the line is not really descriptive

Comment (otyshchenko1) [reply ololo.txt]: why do you think so?

**Review by aanisov — APPROVED** [review]

**Pull request closed without merging**